draft-ZIP-0226.rst

```diff
@@ -27,27 +27,33 @@ The term "Orchard" in this document is to be interpreted as described in ZIP 224
 
 We define the following additional terms:
 
-- Asset: A type of note that can be transferred on the Zcash block chain. ZEC is the default (and currently the only defined) asset for the Zcash mainnet.
-- Zcash Shielded Asset: an asset with issuance defined on the Zcash block chain, and, for now, belonging to the Orchard anonymity pool.
-- Wrapped Asset: a ZSA asset with native issuance defined outside the Zcash block chain.
+- Asset: A type of note that can be transferred on the Zcash block chain.
+
+  - ZEC is the default (and currently the only defined) asset for the Zcash mainnet.
+
+- Zcash Shielded Asset: an Asset with issuance defined on the Zcash block chain, and, for now, belonging to the Orchard shielded pool.
+- Wrapped Asset: a Zcash Shielded Asset with native issuance defined outside the Zcash block chain.
+- Native Asset: a Zcash Shielded Asset created uniquely within the Zcash block chain.
+- Split Note: a Zcash Shielded Asset spend note that is used within an Action to ensure the output note of that Action is of a specific type. The Split Note is not spent in the transaction.
+- Split Action: a Zcash Shielded Asset Action that contains a Split Note.
 
 Abstract
 ========
 
 ZIP 226 and ZIP 227 propose in conjunction the Zcash Shielded Assets (ZSA) protocol, which is an extension of the
-Orchard protocol that enables the creation, transfer and burn of custom assets on the Zcash chain. The creation of such assets is defined
+Orchard protocol that enables the creation, transfer, and burn of custom assets on the Zcash chain. The creation of such assets is defined
 in ZIP 227 [#zip-0227]_. The transfer and burn of such assets is defined in ZIP 226 [#zip-0226]_.
 
 Motivation
 ==========
 
-The current Orchard protocol does not support custom assets, which is an important aspect for the proper evolution of the Zcash ecosystem. Enabling multi-asset support on the Zcash chain will open the door for a host of applications and enhance the ecosystem with application developers and asset custody institutions for issuance and bridging purposes.
+The current Orchard protocol does not support custom assets. Enabling multi-asset support on the Zcash chain will open the door for a host of applications and enhance the ecosystem with application developers and asset custody institutions for issuance and bridging purposes.
```

Comment:
```suggestion
The current Orchard protocol does not support custom assets. Enabling multi-asset support on the Zcash chain will open the door for a host of applications, and enhance the ecosystem with application developers and asset custody institutions for issuance and bridging purposes.
```

Reply:
Suggestion incorporated in [PR#649](https://github.com/zcash/zips/pull/649).